src/Aspire.Dashboard/Components/Pages/Token.razor

```diff
@@ -0,0 +1,40 @@
+﻿@page "/token"
+@layout EmptyLayout
+@attribute [AllowAnonymous]
+
+<PageTitle>Token!</PageTitle>
```

Comment:
I didn't touch the page title here yet since I had no great ideas on what it should say. @kvenkatrajan @leslierichardson95 ?

Reply:
I changed this to ".NET Aspire Token Validation" at @leslierichardson95 's suggestion, but we're still open to feedback as to what this should say.